src/services/print.js

```diff
@@ -550,6 +549,19 @@ ngeo.Print.prototype.getReportUrl = function(ref) {
 
 
 /**
+ * Get the print capabilities from MapFish Print.
+ * @param {angular.$http.Config=} opt_httpConfig $http config object
+ * @return {Object}
```

Comment:
I've just realized that the return type is not correct. It should be `angular.$http.HttpPromise`.